lib/mbta.js

```diff
@@ -1,12 +1,43 @@
 'use strict'
 
-// Code here.
+/*
+Create a function in lib/mbta.js to calculaste the number of stops between stations on the "MBTA".
+Write additional functions used by this function as needed.
+
+The function takes the line and stop that a rider is getting on at and the line and stop that a rider
+is getting off at and returns the total number of stops for the trip.
+There are 3 subway lines:
+
+All 3 subway lines intersect at Park Street, but there are no other intersection points. Some of this MBTA is
+fictionalized.
+*/
 
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
 
+  //stops.orange.indexOf('Back Bay')
+  let station2 = stops.endLine.indexOf(endStation)
```

Comment:
You have the right idea here, but you actually need to reference the `endLine` and `startLine` using bracket notation rather than dot notation. So something like `let station2 = stops[endLine].indexOf(endStation)`. If you want, we can go over dot vs. bracket notation.